tests/unit/mixins/page-speed-mixin-test.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import PageSpeedMixinMixin from 'ember-new-relic/mixins/page-speed-mixin';
+import { module, test } from 'qunit';
+
+module('Unit | Mixin | page speed mixin');
+
+// Replace this with your real tests.
+test('it works', function(assert) {
+  let PageSpeedMixinObject = Ember.Object.extend(PageSpeedMixinMixin);
```

Comment:
Maybe if you mixin `Ember.Evented` you can call `subject.trigger('didTransition')` and it should trigger your function.